src/python/foglamp/core/service_registry/service_registry.py

```diff
@@ -133,18 +146,23 @@
             "type": service._type,
             "address": service._address,
             "port": service._port,
```

Comment:
Changing this to servicePort is changing the meaning of port for existing code and also means it now is not in line with the specification. If we are going to break the existing code we should make this servicePort and not port and then update the uses of this and the documentation.

Reply:
**.port** should be clear itself in terms of service's port. but if we want to change with some diff name, please open the ticket and we can change to that word